components/develop/modules/ROOT/pages/hubs/gaming.adoc

```diff
@@ -3,4 +3,7 @@
 include::partial$what-is-a-skale-hub.adoc[]
 
 == Overview
-include::partial$hub-gaming.adoc[]
\ No newline at end of file
+
+include::partial$hub-gaming.adoc[]
+
+To better understand your game integration with Nebula Hub go to xref:./gaming/integration-steps.adoc[Integration Steps]. 
```

Comment:
Suggestion: 

To learn about building your game on SKALE or migrating an existing game to SKALE, see xref:./gaming/integration-steps.adoc[Integration Steps for Nebula].